src/ResourceManager/Insights/Commands.Insights/Alerts/AddAzureRmAlertRuleCommandBase.cs

```diff
@@ -77,11 +77,11 @@ public abstract class AddAzureRmAlertRuleCommandBase : ManagementCmdletBase
         /// </summary>
         protected override void ProcessRecordInternal()
         {
-            WriteWarning("This output of this cmdlet will change in the next release to return the updated or newly created object.");
+            WriteWarning("The output of this cmdlet will change in the next release to return a single object, not a list, that includes the updated or newly created object.");
```

Comment:
Yeap, I did not want to break everything at the same time. It would take many days to explain all the changes. :)
I would change this next possible release.